resources/create_icns.py

```
@@ -1,128 +1,63 @@
+    for size, scale, filename in ICONSET_SLOTS:
+        pixels = size * scale
+        source = png_by_size.get(pixels)
+        if source is None:
+            raise KeyError(
+                f"no rendered PNG for {pixels}px (slot {filename}); "
+                f"have {sorted(png_by_size)}"
+            )
+        shutil.copyfile(source, iconset_dir / filename)
+        print(f"  - {filename} ({pixels}x{pixels}) <- {Path(source).name}")
 
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)